### PR TITLE
Sms #185

### PR DESCRIPTION
**Pull request by kuanb**

### CODE REVIEW

**Comment by kuanb** [comment]
still working on tests... went brain dead and just went ahead without them. i know, terrible.

File: app/controllers/twilio.js [hunk 1]
```diff
@@ -1,114 +1,87 @@
...
+            let d1 = new Date(lastInboundDate);
+            let d2 = new Date();
+            let timeLapsed = Math.round((d2.getTime() - d1.getTime()) / (3600*1000));
+
+            // If it's been more than 4 hours let's communicate
+            if (timeLapsed > 1) {
+              let dayOfWeek = d2.getDay();
+              if (dayOfWeek == 0 || dayOfWeek == 6) {
+                content = `Message received. Because it 
+                          is the weekend, your case manager may not
```
Comment (maxmcd) [inline app/controllers/twilio.js]: As much as I love the multiline quotes I think the line breaks will be preserved in the sent text message here and look strange to the end user

Comment (kuanb) [inline app/controllers/twilio.js]: Roger

**Comment by kuanb** [comment]
I'm gonna wait for a LGTM from you.. But this is "ok" at this point. Still work to be done, still missing tests, but I am about out of steam for the evening. Might do a little more, but that's it.

**Pull request merged**